lib/index.js

```diff
@@ -400,21 +402,21 @@ var MtSvgLines = function (_React$Component) {
 }(_react2.default.Component);
 
 MtSvgLines.propTypes = {
-  className: _react.PropTypes.string, // custom CSS class (applied to svg elem)
-  animate: _react.PropTypes.oneOfType([// external animation trigger
-  _react.PropTypes.string, // - pass a unique string or true to (re)start animation
-  _react.PropTypes.number, // - pass a number to specify delay before the animation begins (ms)
-  _react.PropTypes.bool // - pass false (or omit) to draw static SVG (no animation)
+  className: PropTypes.string, // custom CSS class (applied to svg elem)
+  animate: PropTypes.oneOfType([// external animation trigger
+  PropTypes.string, // - pass a unique string or true to (re)start animation
+  PropTypes.number, // - pass a number to specify delay before the animation begins (ms)
+  PropTypes.bool // - pass false (or omit) to draw static SVG (no animation)
   ]),
-  duration: _react.PropTypes.number, // total anim duration (ms)
-  stagger: _react.PropTypes.number, // delay between start times of each path (percentage)
-  timing: _react2.default.PropTypes.oneOf([// easing type
+  duration: PropTypes.number, // total anim duration (ms)
+  stagger: PropTypes.number, // delay between start times of each path (percentage)
+  timing: PropTypes.oneOf([// easing type
```

Comment:
This is the only one I'm not 100% sure correct, but I tested it locally and it seems to work fine (plus the default is defined in default props, so should be good regardless).